File: AndroidDSD/src/com/example/androiddsd/entidades/Remesa.java
package com.example.androiddsd.entidades;

import java.io.Serializable;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Remesa implements Serializable{
		
	private static final long serialVersionUID = -8107086530049997467L;
	
	public static final String NUMERO = "Numero";
	public static final String PIN = "PIN";
	public static final String FECHA = "Fecha";
	public static final String AGENCIA = "Agencia";
	public static final String MONEDA = "Moneda";
	public static final String MONTO = "Monto";
	public static final String ESTADO = "Estado";
		
	@Expose
	@SerializedName(NUMERO)
	private int numero;
	
	@Expose
	@SerializedName(PIN)
	private int pin;
	
	@Expose
	@SerializedName(FECHA)
	private String fecha;
	
	@Expose
	@SerializedName(AGENCIA)
	private Agencia agencia;
	
	@Expose
	@SerializedName(MONEDA)
	private Moneda moneda;
	
	@Expose
	@SerializedName(MONTO)
	private double monto;

	@Expose
	@SerializedName(ESTADO)
	private String estado;
	
	public int getNumero() {
		return numero;
	}

	public void setNumero(int numero) {
		this.numero = numero;
	}

	public int getPin() {
		return pin;
	}

	public void setPin(int pin) {
		this.pin = pin;
	}

	public String getFecha() {
		return fecha;
	}

	public void setFecha(String fecha) {
		this.fecha = fecha;
	}

	public Agencia getAgencia() {
		return agencia;
	}

	public void setAgencia(Agencia agencia) {
		this.agencia = agencia;
	}

	public Moneda getMoneda() {
		return moneda;
	}

	public void setMoneda(Moneda moneda) {
		this.moneda = moneda;
	}

	public double getMonto() {
		return monto;
	}

	public void setMonto(double monto) {
		this.monto = monto;
	}

	public String getEstado() {
		return estado;
	}

	public void setEstado(String estado) {
		this.estado = estado;
	}
	
}

File: ProyectoDSD/DSD.Entidades/Remesa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Runtime.Serialization;

[DataContract]
public class Remesa
{
    [DataMember]
    public int Numero { get; set; }
    [DataMember]
    public int PIN { get; set; }
    [DataMember]
    public String Fecha { get; set; }
    [DataMember]
    public Agencia Agencia { get; set; }
    [DataMember]
    public Moneda Moneda { get; set; }
    [DataMember]
    public decimal Monto { get; set; }
    [DataMember]
    public String Estado { get; set; }
}

[DataContract]
public class HoraExcedidaException {

    [DataMember]
    public string DetalleError;

    public HoraExcedidaException(string error)
    {
        DetalleError = error;
    }
}
